src/edu/psu/sweng500/team8/gui/CellGUI.java

```diff
@@ -0,0 +1,513 @@
+		pencilMarkDisplay[2][0]
+				.setHorizontalAlignment(javax.swing.SwingConstants.CENTER);
+		pencilMarkDisplay[2][0].setText("7");
+		pencilMarkDisplay[2][0].setMaximumSize(new java.awt.Dimension(
+				NUMBER_SIZE, NUMBER_SIZE));
+		pencilMarkDisplay[2][0].setMinimumSize(new java.awt.Dimension(
+				NUMBER_SIZE, NUMBER_SIZE));
+		pencilMarkDisplay[2][0].setPreferredSize(new java.awt.Dimension(
+				NUMBER_SIZE, NUMBER_SIZE));
+		pencilMarkDisplay[2][0].setVisible(false);
+		
+		gridBagConstraints = new java.awt.GridBagConstraints();
+		gridBagConstraints.gridx = 0;
+		gridBagConstraints.gridy = 2;
+		gridBagConstraints.anchor = java.awt.GridBagConstraints.NORTHWEST;
+		// gridBagConstraints.insets = new java.awt.Insets(0, 1, 1, 0);
+		pencilMarkDisplayCell.add(pencilMarkDisplay[2][0], gridBagConstraints);
+
+		pencilMarkDisplay[2][1] = new JLabel();
+		pencilMarkDisplay[2][1]
+				.setHorizontalAlignment(javax.swing.SwingConstants.CENTER);
+		pencilMarkDisplay[2][1].setText("8");
+		pencilMarkDisplay[2][1].setMaximumSize(new java.awt.Dimension(
+				NUMBER_SIZE, NUMBER_SIZE));
+		pencilMarkDisplay[2][1].setMinimumSize(new java.awt.Dimension(
+				NUMBER_SIZE, NUMBER_SIZE));
+		pencilMarkDisplay[2][1].setPreferredSize(new java.awt.Dimension(
+				NUMBER_SIZE, NUMBER_SIZE));
+		pencilMarkDisplay[2][1].setVisible(false);
+		
+		gridBagConstraints = new java.awt.GridBagConstraints();
+		gridBagConstraints.gridx = 1;
+		gridBagConstraints.gridy = 2;
+		gridBagConstraints.anchor = java.awt.GridBagConstraints.NORTHWEST;
+		// gridBagConstraints.insets = new java.awt.Insets(0, 0, 1, 0);
+		pencilMarkDisplayCell.add(pencilMarkDisplay[2][1], gridBagConstraints);
+
+		pencilMarkDisplay[2][2] = new JLabel();
+		pencilMarkDisplay[2][2]
+				.setHorizontalAlignment(javax.swing.SwingConstants.CENTER);
+		pencilMarkDisplay[2][2].setText("9");
+		pencilMarkDisplay[2][2].setMaximumSize(new java.awt.Dimension(
+				NUMBER_SIZE, NUMBER_SIZE));
+		pencilMarkDisplay[2][2].setMinimumSize(new java.awt.Dimension(
+				NUMBER_SIZE, NUMBER_SIZE));
+		pencilMarkDisplay[2][2].setPreferredSize(new java.awt.Dimension(
+				NUMBER_SIZE, NUMBER_SIZE));
+		pencilMarkDisplay[2][2].setVisible(false);
+		
+		gridBagConstraints = new java.awt.GridBagConstraints();
+		gridBagConstraints.gridx = 2;
+		gridBagConstraints.gridy = 2;
+		gridBagConstraints.anchor = java.awt.GridBagConstraints.NORTHWEST;
+		// gridBagConstraints.insets = new java.awt.Insets(0, 0, 1, 1);
+		pencilMarkDisplayCell.add(pencilMarkDisplay[2][2], gridBagConstraints);
+	}
+
+
+	private void markGivenCell() {
+		HighlightPainter painter = new DefaultHighlighter.DefaultHighlightPainter(
+				Color.green);
+
+		try {
+			// (JN): What is the significance of 0,3?
+			this.numberInputField.getHighlighter().addHighlight(0, 3, painter);
+		} catch (BadLocationException e) {
+			e.printStackTrace();
+			/* TODO - ignore or handle ? */
+		}
+	}
+
+	private void selectCell(CellGUI selectedCell) {

```

Comment:
when I was getting selectedSell method, it wasn't always 'this.' but it
might have happened to be always 'this' now. I will look into it and see
what I can do.

On Sun, Jul 12, 2015 at 10:30 PM, Jeremy Nelson <notifications@github.com>
wrote:

> In src/edu/psu/sweng500/team8/gui/CellGUI.java
> <https://github.com/garnryang/Team8/pull/13#discussion_r34430032>:
>
> > +
> > +
> > +	private void markGivenCell() {
> > +		HighlightPainter painter = new DefaultHighlighter.DefaultHighlightPainter(
> > +				Color.green);
> > +
> > +		try {
> > +			// (JN): What is the significance of 0,3?
> > +			this.numberInputField.getHighlighter().addHighlight(0, 3, painter);
> > +		} catch (BadLocationException e) {
> > +			e.printStackTrace();
> > +			/* TODO - ignore or handle ? */
> > +		}
> > +	}
> > +
> > +	private void selectCell(CellGUI selectedCell) {
>
> selectedCell is always "this" -- why not just move this logic to
> selectCell() ?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/garnryang/Team8/pull/13/files#r34430032>.
>